android/buildAndroidBOINC-CI.sh

```diff
@@ -91,18 +95,18 @@ while [ $# -gt 0 ]; do
 done
 
 if [ "x$cache_dir" != "x" ]; then
-    if isPathCanonical "$cache_dir" && [ "$cache_dir" != "/" ]; then
-         PREFIX="$cache_dir"
-     else
-         echo "cache_dir must be an absolute path without ./ or ../ in it"
-         exit 1
-     fi
+    if  ! ( isPathCanonical "$cache_dir" && [ "$cache_dir" != "/" ] ); then
```

Comment:
`&&` should be changed to `||` (`and` to `or`) otherwise statement is not correct, If I understand it correctly